src/hooks/useIndexOfLastVisibleChild.mdx

```diff
@@ -25,10 +25,10 @@ of the children until they exceed the width of the container.
     pointerEvents: 'none',
     visibility: 'hidden',
   };
-  const containerElementRef = React.useRef(null);
+  const [containerElementRef, setContainerElementRef] = React.useState(null);
```

Comment:
This is technically a breaking change for consumers, but I don't see any usage of this hook in Usage Insighhts table, so I think it should be okay to release it as a fix.

The reason we need to use callback ref is to trigger a re-render once the ref is attached to the DOM element, previously this re-redned was caused by `useWindowSize` hook that was used in `useIndexOfLastVisibleChild` before this PR.

Reply:
[inform] Yeah, it looks like there aren't any consumers of this hook from Paragon directly, but there are (nearly) duplicate functions that exist within frontend-app-learning ([source](https://github.com/openedx/frontend-app-learning/blob/600f5b4dff2f1f79e073a1ad8b33eadb8c6ee281/src/generic/tabs/useIndexOfLastVisibleChild.js#L32)) and frontend-app-discussions ([source](https://github.com/openedx/frontend-app-discussions/blob/b5d036a54d983241a5898fed735937f348754901/src/components/NavigationBar/tabs/useIndexOfLastVisibleChild.js#L33)). They were likely implemented before we implemented the responsive `Tabs` in Paragon.

I agree this should probably be OK to release a fix.